packages/react-components/tsconfig.json

```diff
@@ -1,22 +1,17 @@
 {
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["ES2016", "dom"],
```

Comment:
looks like `ES2015` won't be enough as we are using ES2015+ features (`Array.prototype.includes` for instance etc ) 

We should update migration generator to this change as well - cc @ling1726